src/FileSystem-Core.package/Path.class/class/from.delimiter..st

```diff
@@ -5,8 +5,10 @@ from: aString delimiter: aDelimiterCharacter
 	aString isEmpty
 		ifTrue: [ ^ self root ].
 	
-	pathClass :=  ((self isAbsoluteUnixPath: aString) or: [self isAbsoluteWindowsPath: aString]) 
+	pathClass :=  ((self isAbsolutePath: aString delimiter: aDelimiterCharacter) or: 
+							[self isAbsoluteWindowsPath: aString]) 
 		ifTrue: [ AbsolutePath ]
 		ifFalse:[ RelativePath ].
 	
-	^ pathClass withAll: (pathClass canonicalizeElements: (aDelimiterCharacter split: aString))
\ No newline at end of file
+	^ pathClass withAll: ((aDelimiterCharacter split: aString) select: 
```

Comment:
Hi @gcotelli,

Thanks for reviewing this, and my apologies for the slow reply (I wasn't subscribed to the issue :-().

Just FYI, the justification for removing the canonicalisation is described in:
http://forum.world.st/FileReference-gt-gt-and-path-canonicalisation-td4952575.html

However it isn't part of this patch.  I obviously forgot to check that I wasn't modifying a method that had already been modified as part of my other work.

Great catch, and thanks for letting me know.

Pavel, my apologies also, but I think you need to make this change as you own the branch that generated the pull request.  If there is a way I can do it, please let me know so I can save time in future.

I'll make the change, test it, and post the modified method in a follow-up comment (real soon now).

Thanks again,
Alistair